src/pages/messages/components/NviCandidateActionPanel.tsx

```diff
@@ -0,0 +1,77 @@
+import { TabContext, TabList, TabPanel } from '@mui/lab';
+import { Divider, Paper, styled, Tab, Typography } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFetchRegistration } from '../../../api/hooks/useFetchRegistration';
+import { useFetchRegistrationTickets } from '../../../api/hooks/useFetchRegistrationTickets';
+import { NviCandidate } from '../../../types/nvi.types';
+import { getIdentifierFromId } from '../../../utils/general-helpers';
+import { LogPanel } from '../../public_registration/LogPanel';
+import { NviApprovals } from './NviApprovals';
+import { NviCandidateActions } from './NviCandidateActions';
+
+interface NviCandidateActionPanelProps {
+  nviCandidate: NviCandidate;
+  nviCandidateQueryKey: string[];
+}
+
+enum TabValue {
+  Dialogue = 'Dialogue',
+  Log = 'Log',
+}
+
+const StyledTabPanel = styled(TabPanel)({
+  padding: 0,
+});
+
+export const NviCandidateActionPanel = ({ nviCandidate, nviCandidateQueryKey }: NviCandidateActionPanelProps) => {
+  const { t } = useTranslation();
+  const [tabValue, setTabValue] = useState(TabValue.Dialogue);
+
+  const ticketsQuery = useFetchRegistrationTickets(nviCandidate.publicationId);
+  const registrationQuery = useFetchRegistration(getIdentifierFromId(nviCandidate.publicationId));
+
+  const tickets = ticketsQuery.data?.tickets ?? [];
+  const periodStatus = nviCandidate?.period.status;
+  const pointsSum = nviCandidate?.approvals.reduce((acc, curr) => acc + curr.points, 0) ?? 0;
+
+  return (
+    <Paper
+      elevation={0}
+      sx={{
+        gridArea: 'nvi',
+        bgcolor: tabValue === TabValue.Dialogue ? 'nvi.light' : 'registration.light',
+        height: 'fit-content',
+        minHeight: { sm: '85vh' },
+        display: 'flex',
+        flexDirection: 'column',
+      }}>
+      <TabContext value={tabValue}>
+        <TabList
+          onChange={(_, newValue: TabValue) => setTabValue(newValue)}
+          sx={{ bgcolor: 'primary.main', color: 'primary.contrastText', px: '0.5rem' }}
```

Comment:
Stylingen her er stort sett basert på den "vanlige" ActionPanel, bare at jeg har tatt i bruk flere MUI-komponenter for tabs, som hevder å fikse mer rundt a11y ut av boksen. Disse ligger i `@mui/lab`